src/Modulo2.jl

```diff
@@ -323,11 +323,13 @@ end
 *(c::Number, a::ZZ2Array) = iszero(c) ? zero(a) : copy(a)
 # end of the list
 
-function *(a::ZZ2Matrix, b::ZZ2Vector)
+*(a::ZZ2Matrix, b::ZZ2Vector) = mul!(ZZ2Vector(undef, size(a,1)), a, b)
```

Comment:
```suggestion
*(a::ZZ2Matrix, b::ZZ2Vector) = mul!(ZZ2Vector(undef, size(a, 1); init = false), a, b)
```
This way we skip setting up the padding.